galaxy_ng/app/models/container.py

```diff
@@ -23,3 +23,39 @@ class ContainerDistroReadme(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True, null=True)
     text = models.TextField(blank=True)
+
+
+class ContainerRegistryRemote(
+    pulp_models.Remote,
+    LifecycleModelMixin,
```

Comment:
is `LifecycleModelMixin` required? I think all the pulp base classes inherit from `LifecycleModel`